fiaas_deploy_daemon/specs/v3/factory.py

```diff
@@ -47,7 +47,7 @@ def __call__(self, name, image, teams, tags, app_config, deployment_id, namespac
             namespace=namespace,
             name=name,
             image=image,
-            replicas=lookup["replicas"]["maximum"],
+            replicas=lookup["replicas"]["minimum"],
```

Comment:
is this field used now?

Reply:
Yes, i.e. [here](https://github.com/fiaas/fiaas-deploy-daemon/blob/master/fiaas_deploy_daemon/deployer/kubernetes/autoscaler.py#L45) but maybe it can be replaced by `app_spec.autoscaler.min_replicas`

Reply:
that [line](https://github.com/fiaas/fiaas-deploy-daemon/blob/master/fiaas_deploy_daemon/deployer/kubernetes/autoscaler.py#L45) needs to be `app_spec.autoscaler.max_replicas` indeed, because you're setting replicas to min here. If that is the only place, I'd remove it from the app spec to avoid confusion on its meaning. WDYT?

Reply:
Oh right, will do that :+1: